lesson35/link-on-playgtound.md

```diff
@@ -0,0 +1,5 @@
+1 lesson task 
```

Comment:
```typescript
//4
function usersHasAnimals(arr: Array<{ animals: [] }>) {
    return arr.filter(user => user.animals);
}
```
array length should be checked as well because empty array can be presented